properties.gradle

```diff
@@ -0,0 +1,2 @@
+ext.libraryVersionCode=100128
+ext.libraryVersionName='0.1.28'
```

Comment:
I wouldn't name this file properties.gradle. It gives the false impression that this is a [gradle properties file](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties) while it is not.
Also, the `ext` part is not used and I guess we don't need both code and name, do we ? 
What about a simple flat file named `version` and just containing the versionCode ? `128` ?